### DISCUSSION TITLE
UniTask support for When

### DISCUSSION THREAD

**Discussion by slimshader** [codewriter-packages/UniMob#19] [Ideas]
Hi, would be cool to have support for UniTask as a "promise" return type for When(). Possibly configurable via define?

**Comment by slimshader** [comment]
bump

**Comment by vanifatovvlad** [comment]
Yes, that would be cool. However, to implement the allocation-free method, it is also necessary to rework the creation of a temporary lifetime and reaction. This is a fairly large rework for which I do not have time yet

**Comment by slimshader** [comment]
This is implemented by https://github.com/codewriter-packages/UniMob/blob/master/Runtime/Atom.UniTask.When.cs right?
